tests/unit/btree_stress_test.c

```diff
@@ -327,8 +323,8 @@ insert_tests(cache           *cc,
          ASSERT_TRUE(FALSE, "Failed to insert 4-byte %ld\n", i);
       }
    }
-   platform_free(heap_id, keybuf);
-   platform_free(heap_id, msgbuf);
+   platform_free(hid, keybuf);
```

Comment:
In the old code (not seen in this diff block), allocation was done using `hid` but free is being done using `heap_id`.

For current scheme in /main, this is a non-issue. But while supporting new under-development feature for different memory models, this mixed usage becomes problematic. It could lead to memory leaks, too!